scss/_patterns_navigation.scss

```diff
@@ -95,10 +135,12 @@ $lightness-threshold: 70;
     &__row {
       @extend %fixed-width-container;
       @extend %navigation-row;
+      @extend %vf-reset-horizontal-padding;
 
       &--full-width {
         @extend %vf-grid-container-padding;
         @extend %navigation-row;
+        @extend %vf-reset-horizontal-padding;
```

Comment:
If we need to reset paddings to 0, why do we set them 2 lines above with `%vf-grid-container-padding`?